docs/user/access-control.md

```diff
@@ -17,7 +17,7 @@ Two of the most common authentication strategies for providing a token identifyi
 
 ### OpenID Connect authentication
 
-The most common and secure authentication for users to authenticate with the cluster (and therefore Kubeapps) is to use the built-in Kubernetes support for OpenID Connect. In this setup your clusters trust an OAuth2 provider such as Azure Active Directory, Google OpenID Connect or your own installation of the Dex auth provider. You can read more about [using an OIDC provider with Kubeapps](../using-an-OIDC-provider.md).
+The most common and secure authentication for users to authenticate with the cluster (and therefore Kubeapps) is to use the built-in Kubernetes support for OpenID Connect. In this setup your clusters trust an OAuth2 provider such as Azure Active Directory, Google OpenID Connect or your own installation of the Dex auth provider. You can read more about [using an OIDC provider with Kubeapps](./using-an-OIDC-provider.md).
```

Comment:
Hmm, so my guess is that @andresmgot updated this so it works on the new docs site, but you're right, it needs to be able to work in the GH docs also - I assumed that with the change for the docs site it worked for both already.

From a brief look, the issue is that on the docs site, this page (access-control.md) is presented with a nice clean URL: https://kubeapps.com/docs/access-control/ but this requires the `../` to get back to `/docs/`. We need the links to work on both - it may be that an easy option is to not use pretty URLs on the docs site (ie. if it was `/docs/access-control` without the slash, then `./using-an-OIDC-provider.md` would translate, I think? Or perhaps there's a better way with the docs config. Let's see what @andresmgot thinks - but I'd avoid landing this change until we find out his intention.

Reply:
Yes, the thing is that the trailing slash `/` cannot be removed (or at least I am not aware  of a way of doing that). In any case, we already have a hook to replace links so we can just do `./` --> `../` when rendering the docs in kubeapps.com.

Summary: this change is fine, sorry for not noticing when working on this.

Reply:
> so we can just do `./` --> `../` when rendering the docs in kubeapps.com

Maybe, though it'd be better if links just worked between GH and the docs site. I don't remember, but think you mentioned that you chose middleman for the docs site? Does this help? https://gist.github.com/mxhold/00b076c996d6d74401c1

Reply:
thanks, will try that as well